openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -115,14 +180,31 @@ def load_casting(project_name, shot_name) -> Set[OpenpypeContainer]:
             else:
                 subset_name = "rigMain"
                 loader_name = "LinkRigLoader"
-            try:
-                container, _datablocks = load_subset(
-                    project_name, actor["asset_name"], subset_name, loader_name
+
+            # Download subset
+            representation = download_subset(
+                project_name, actor["asset_name"], subset_name
+            )
+            if representation:
+                representations.append(
+                    download_subset(
+                        project_name, actor["asset_name"], subset_name
+                    )
                 )
```

Comment:
Can't this be `representations.append(representation)`?

Reply:
You're right, forgot simplification